src/appComponents/common/ApiJsonFactory.js

```diff
@@ -15,31 +15,30 @@ var baseUrl = (config.use_testApi ? "testapi/" : config.apiBaseGetUrl) + "event/
                 let endpoint = "";
 
               if ($apiEndpoint === "event" && !config.use_testApi) {
-                 
                   endpoint = baseUrl;
                 
               } 
               else {
-                 
                   endpoint = baseUrl + "/" + $apiEndpoint;
                 
               }
               const deferred = $q.defer(),
-
               httpPromise = $http.get(endpoint);
-              httpPromise.then(function (response) {
 
+              httpPromise.then(function (response) {
                   deferred.resolve(response);
 
               }, function (error) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["error" is defined but never used](https://www.codacy.com/app/triplez/open-event-webapp/file/2444008392/issues/source?bid=3233857&fileBranchId=3245777#l31)